File: ass1/fourargs.cpp
//Copyright 2018 Vaibhav Bansal vbansal@bu.edu

#include<iostream>

int main(int argc, char *argv[])
{	
	//extern ostream cout;
	//extern ostream cerr;

	int i=1;
	for (i=1;i<argc;i++)
		{if (i<=4)
			std::cout<<argv[i]<<std::endl;
		else
			std::cerr<<argv[i]<<std::endl;
		}
return 0;
}

File: ass1/fourargs.py
#Copyright 2018 Vaibhav Bansal vbansal@bu.edu

import sys
i=1
while (i<len(sys.argv)):
	if i<5:
		sys.stdout.write(sys.argv[i]+"\n")
		
	else:
		sys.stderr.write(sys.argv[i]+"\n")
		
	i=i+1
